features/fields_extensie.feature

```diff
@@ -1,126 +1,570 @@
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Minimale combinatie van parameters moet worden opgegeven.                                                   |
+      | status   | 400                                                                                                         |
+      | detail   | De parameter fields mag niet leeg zijn.                                                                     |
+      | code     | paramsCombination                                                                                           |
+      | instance | /personen                                                                                                   |
+      En heeft de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                  |
+      | required | fields        | Parameter is verplicht. |
+
+  Rule: De Fields parameter mag alleen valide en unieke attribuutnamen bevatten.
+    Scenario: Fields parameter is leeg
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999994086                       |
+      | fields              |                                 |
+      Dan heeft de response de volgende gegevens
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Een of meerdere parameters zijn niet correct.                                                               |
+      | status   | 400                                                                                                         |
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En heeft de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                                      |
+      | pattern  | fields        | Waarde voldoet niet aan patroon {pattern}.  |
+
+    Scenario: Fields parameter met attribuutnaam die niet bestaat
```

Comment:
Hier wordt over attribuutnaam gesproken, maar in de response krijgt je propertynaam. attribuutnaam hernoemen tbv consistentie?